packages/commands/src/private-apis.js

```diff
@@ -19,4 +20,5 @@ export const privateApis = {};
 lock( privateApis, {
 	useCommand,
 	useCommandLoader,
+	store,
```

Comment:
Should we export the whole store here?

Reply:
I think so like other packages, I just made it private to start with.

Reply:
I think it's fine to start with. I couldn't see any other package locking the whole store though. We use `registerPrivateActions` and the rest.